block/block.go

```diff
@@ -48,6 +50,16 @@ func (m *Manager) applyBlock(block *types.Block, commit *types.Commit, blockMeta
 			return fmt.Errorf("save block: %w", err)
 		}
 
+		gossipedBlock := p2p.BlockData{Block: *block, Commit: *commit}
```

Comment:
for cleanliness I'd suggest moving this code part to a `saveP2PBlock` method or some other name which makes sense an add a godoc to this function. 

also we why use `context.TODO()`? s

Reply:
moved to new func